src/artemisa/component.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import { connect } from 'react-redux'
 
-import { State, isInStateSlot } from '../core/model'
+import { State } from '../core/model'
 import { isReceive } from '../core/actions'
+import { hasValueInKey } from '../util/object'
 import { shouldFetch } from '../core/reducer'
-import { isFunction, trueFn, identity } from '../util/object'
+import { isFunction, trueFn, identity } from '../util/function'
 export const ARTEMISA = 'ARTEMISA'
 
-export const isArtemisaType = type => type.indexOf(ARTEMISA) === 0
+export const isArtemisaType = type => !!type.match('ARTEMISA')
```

Comment:
mm.. I'm not sure this is correct.
The intention here is to match any string that **starts with** ARTEMISA.
So, maybe 
```
!!type.match('^ARTEMISA.*')
```
or something like that ?